Dockerfile

```diff
@@ -1,11 +1 @@
-FROM instrumentisto/rsync-ssh:alpine3.13-r4
-LABEL "com.github.actions.name"="GitHub Action for WP Engine Site Deploy"
-LABEL "com.github.actions.description"="An action to deploy your repository to WP Engine via the SSH Gateway"
-LABEL "com.github.actions.icon"="upload-cloud"
-LABEL "com.github.actions.color"="blue"
-LABEL "repository"="http://github.com/wpengine/github-action-wpe-site-deploy"
-LABEL "maintainer"="Alex Zuniga <alex.zuniga@wpengine.com>"
-RUN apk add bash php
-ADD entrypoint.sh /entrypoint.sh
-ADD exclude.txt /exclude.txt
-ENTRYPOINT ["/entrypoint.sh"]
+FROM wpengine/gha:v1
```

Comment:
Is this still necessary? Can we not simply point to the newly uploaded image in DockerHub in our action.yml?

Reply:
Ooo maybe. I'll try it next sprint.

Reply:
Switched that up in [d364851](https://github.com/wpengine/github-action-wpe-site-deploy/pull/36/commits/d364851dd1790471d944cb0d5e1b060227f4ae74) --- I think that shaved off another 3 seconds!
![cicd-24-load-docker-image-directly](https://user-images.githubusercontent.com/26469099/184727097-53559962-e7ac-43be-9750-bc0ae12d8d8b.png)